tests/foreman/ui/test_repositories.py

```diff
@@ -96,3 +96,49 @@ def test_positive_override_custom_products_on_existing_host(
         repo = session.host_new.get_repo_sets(rhel_contenthost.hostname, custom_repo.name)
         assert repo[0]['Repository'] == custom_repo.name
         assert repo[0]['Status'] == 'Enabled'
+
+
+@pytest.mark.rhel_ver_match('[^6]')
+def test_positive_override_custom_products_not_using_select_all(
+    session,
+    default_location,
+    setup_content,
+    rhel_contenthost,
+    target_sat,
+):
+    """Verify that custom products can be easily enabled/disabled on existing host
+    by individually selecting the repositories from the table instead of the select all method
+
+    :id: 9d5c05fb-3683-40a2-91fd-a3aa526b25f4
+
+    :steps:
+        1. Create custom product and upload repository
+        2. Attach to activation key
```

Comment:
I would suggest to move these 2 points  by adding new section `:setup:`  before steps, as setup_content is part of fixture and doing both of the things
```suggestion
    :setup:
        1. Create custom product and upload repository
        2. Attach to activation key
    :setps:
        1. Register Host
        ...
```